lib/rng-browser.js

```diff
@@ -2,25 +2,23 @@
 // browser this is a little complicated due to unknown quality of Math.random()
 // and inconsistent support for the `crypto` API.  We do the best we can via
 // feature-detection
-var rng;
 
-var crypto = global.crypto || global.msCrypto; // for IE 11
-if (crypto && crypto.getRandomValues) {
+if (typeof(crypto) != 'undefined' && crypto.getRandomValues) {
```

Comment:
`!==` ?

Reply:
I'll address this and the other eslint issues you raised in the other PRs in the `eslint` PR.